🐛[bug] 어드민 데일리 질문 추가 시 에러

# 🐞 버그/에러 개요
attempted to assign id from null one-to-one property 발생

# 📝 상황 설명
### 📄  에러 대상
QuestionCategory

### 🕵🏻‍♀️    에러 상황
Caused by: org.hibernate.id.IdentifierGenerationException: attempted to assign id from null one-to-one property [oncog.cogroom.domain.daily.entity.QuestionCategory.category]

### ✅ Resolve TODO
- [ ]  에러 수정

### 📚 Remarks

--- a/src/main/java/oncog/cogroom/domain/admin/service/AdminService.java
+++ b/src/main/java/oncog/cogroom/domain/admin/service/AdminService.java
@@ -132,6 +132,8 @@ public class AdminService extends BaseService {
             List<QuestionCategory> questionCategories = categories.stream()
                     .map(category -> QuestionCategory.builder()
                             .id(new QuestionCategoryId(question.getId(), category.getId()))
+                            .question(question)
+                            .category(category)
                             .build())
                     .toList();
 

